### PR TITLE
Fix issue when user_id is undefined

### PR DESCRIPTION
**Pull request by bojans1005**

### CODE REVIEW

**Review by ferhatelmas — APPROVED** [review]
LGTM

**Comment by ferhatelmas** [comment]
@bojans1005 could you resolve the conflict?

**Comment by bojans1005** [comment]
@ferhatelmas resolved conflict

**Comment by ferhatelmas** [comment]
@bojans1005 thanks, I will delegate @vishalnarkhede for merge.

File: src/channel.ts [hunk 1]
```diff
@@ -1236,7 +1236,7 @@ export class Channel<
       // set the channel as active...
 
       const membersStr = state.members
-        .map((member) => member.user_id)
+        .map((m) => m.user_id || m.user?.id)
```
Comment (virdesai) [inline src/channel.ts:1239]:
```suggestion
        .map((member) => member.user_id || member.user?.id)
```

**Review by vishalnarkhede — APPROVED** [review]

**Pull request merged**